workflow-controller/controller-service/pom.xml

```diff
@@ -67,6 +67,7 @@
         <dependency>
             <groupId>io.hyscale</groupId>
             <artifactId>schema-validator</artifactId>
+            <version>${project.version}</version>
```

Comment:
For version we are using dependency management in parent pom, this is not required here